gravitee-apim-console-webui/src/organization/configuration/identity-providers/org-settings-identity-providers.component.html

```diff
@@ -41,7 +41,7 @@ <h2>Identity Providers</h2>
     <button (click)="onAddIdpClicked()" mat-raised-button color="primary"><mat-icon>add</mat-icon>Add an identity provider</button>
   </div>
 
-  <gio-table-wrapper [length]="tableData.length" (filtersChange)="onFiltersChanged($event)">
+  <gio-table-wrapper [length]="tableUnpaginatedLength" (filtersChange)="onFiltersChanged($event)">
```

Comment:
```suggestion
  <gio-table-wrapper [length]="totalLength" (filtersChange)="onFiltersChanged($event)">
```

Reply:
why not. looks like  one of the first names I put here 😁 

i've change because I didn't want it to be confused with the total length which is not paginated here
and i've add `table` to have same convention on each variable dedicated to table display stuff 🤷‍♂️

Reply:
Understood 🚀